app/helpers/application_helper.rb

```diff
@@ -1,17 +1,22 @@
+# frozen_string_literal: true
+
+# Module to setup the application layout and navbar
+# helpers (this in order to achieve a cleaner code).
```

Comment:
same here

Reply:
https://www.alibabacloud.com/blog/the-art-of-code-comments-does-good-code-need-comments_600184

Reply:
Fixed!